nss-day-cohort-44/nutshell-whistling-wraiths
# Mc edit tasks

# Description
Made it so that you can edit a task.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
git checkout master
git fetch --all
git checkout mc-editTasks
serve

create a new task to be sure that function still works.
Hit the edit button for that task. A form should appear which is prefilled with the current information.
Save the edit. The form should disappear, and your task in the task bar menu should be updated with the new information.
# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
